app/models/course.rb

```diff
@@ -290,6 +293,10 @@ def initially_refreshed?
     refreshed_at.nil?
   end
```

Comment:
This has only been used once in production, which we later predictably regretted. There are probably a whiole bunch of things that this code leaves behind as junk. Feel free to just remove this action (and the corresponding link in the UI).